modules/virt-pxe-booting-with-mac-address.adoc

```diff
@@ -28,30 +28,27 @@ endif::openshift-rosa,openshift-dedicated[]
 apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
-  name: pxe-net-conf
+  name: pxe-net-conf # <1>
 spec:
-  config: '{
-    "cniVersion": "0.3.1",
-    "name": "pxe-net-conf",
-    "plugins": [
-      {
-        "type": "cnv-bridge",
-        "bridge": "br1",
-        "vlan": 1 <1>
-      },
-      {
-        "type": "cnv-tuning" <2>
-      }
-    ]
-  }'
-----
-<1> Optional: The VLAN tag.
-<2> The `cnv-tuning` plugin provides support for custom MAC addresses.
-+
-[NOTE]
-====
-The virtual machine instance will be attached to the bridge `br1` through an access port with the requested VLAN.
-====
+  config: |
+    {
+      "cniVersion": "0.3.1",
+      "name": "pxe-net-conf", <2>
+      "type": "bridge", <3>
+      "bridge": "bridge-interface", <4>
+      "macspoofchk": false, <5>
+      "vlan": 100, <6>
+      "disableContainerInterface": true,
+      "preserveDefaultVlan": false <7>
```

Comment:
```suggestion
  #   "macspoofchk": false, <5>
  #   "vlan": 100, <6>
  #   "disableContainerInterface": true,
  #   "preserveDefaultVlan": false <7>
```

Reply:
Sorry this suggestion corresponds with my comment about commenting out optional configs 🙃

Reply:
@skrthomas IIUC, the content after the `config: |` line is embedded JSON data and I think the JSON specification does not allow comments.

@phoracek Can you please confirm?

Reply:
Ahhh ok I see. I didn't realize that. Thanks for clarifying. It was only a suggestion anyways so no worries :)